samples/extensions/ray_tracing_reflection/ray_tracing_reflection.cpp

```diff
@@ -903,6 +903,26 @@ void RaytracingReflection::build_command_buffers()
 		                      VK_IMAGE_LAYOUT_TRANSFER_SRC_OPTIMAL,
 		                      VK_IMAGE_LAYOUT_GENERAL,
 		                      subresource_range);
+
+		/*
+			Start a new render pass to draw the UI overlay on top of the ray traced image
```

Comment:
Worth adding a comment which says dependencies/barriers/layout-transitions are handled by sub-pass dependencies? Else it might look like a pipeline-barrier is missing.

Reply:
I'm not sure if such a comment would make sense in that place. If so, we would need to add in in all samples where implicit transitions are used.

Reply:
Fair enough; I wasn't sure if this is something the samples are already explicit about or not :)